docs/system_overview/genquery.md

```diff
@@ -1,12 +1,10 @@
-#
-
-## iRODS General Query (GenQuery)
+# iRODS General Query (GenQuery)
 
 GenQuery is the SQL-like syntax used to query the iRODS catalog. It is used throughout the server to accomplish everything required of the system from resource hierarchy resolution to enforcing permissions, as well as directly by users via `iquest` and other clients.
 
-This page describes syntax for **GenQuery** because it has existed for almost as long as iRODS has existed and is used extensively throughout existing iRODS clients and servers. A more modern solution is now available in the iRODS server called **GenQuery2** which will eventually replace the legacy GenQuery implementation.
+This page describes syntax for **GenQuery** because it has existed for almost as long as iRODS has existed and is used extensively throughout existing iRODS clients and servers. A more modern solution is now available in the iRODS server called [**GenQuery2**](#genquery2) which will eventually replace the legacy GenQuery implementation.
```

Comment:
```suggestion
This page describes syntax for **GenQuery** which is used extensively throughout iRODS clients and servers. A more modern solution is now available in the iRODS server called [**GenQuery2**](#genquery2) which will eventually replace the original GenQuery(1) implementation.
```

Reply:
Done.

Used GenQuery1 instead of GenQuery(1) since GenQuery1 is referenced throughout the doc.